### DISCUSSION TITLE
Trouble downloading and converting music to MPEG4

### DISCUSSION THREAD

**Discussion by mackfire** [aclap-dev/video-downloadhelper#257] [Help and Q&A]
### Your question

I am having problems with Video Downloadhelper converting to MPEG4 files.  It worked a few weeks ago and now will only download to MP3 files.  I am running iOS 11.7.10 and downloadhelper 8.2.1.1 and bought a license with still no luck.  Any ideas why it will not convert?  I have tried most of the setting Behavior and More in settings gear.

### Addon & CoApp version and details

I am having problems with Video Downloadhelper converting to MPEG4 files.  It worked a few weeks ago and now will only download to MP3 files.  I am running iOS 11.7.10 and downloadhelper 8.2.1.1 and bought a license with still no luck.  Any ideas why it will not convert?  I have tried most of the setting Behavior and More in settings gear.

**Comment by GrampaWildWilly** [comment]
You should provide a proper problem report.  You can find out how to do that -- and many other useful things -- by clicking the Wiki link at the top of this page. The Wiki page shows a Table of Contents down the right side. You would do well to look at all of the chapters, but for the moment, just go to the chapter "I need help."  You should be able to take it from there.

There were some problems with conversion recently, but it is my understanding that most of those have been cured.  (I do not work for VDH.  I'm just another user.)  Depending on what browser you are using, a bit of information you would provide within a proper problem report, a recent beta could be of some help.  Beta versions is another chapter in the Wiki.

**Reply by mackfire** [reply]
Thanks for the reply.  Not sure how to do a proper problem report, so that is above my pay grade.  I get a side download error in the bottom box and the details as "Invalid audio stream. Exactly one MP3 audio stream is required."

**Reply by GrampaWildWilly** [reply]
At the top of this page there is a link labelled Wiki.  Click that.  That is a user's guide of sorts.  There is a Table of Contents down the right side of the page.  Look for the chapter with the title "I need help."  Click that.  Read what you find & follow the instructions.  That will tell you how to post a proper problem report.  Come on.  That doesn't take a pay grade.  Then you have to give us at least one URL, if that's appropriate.  If you're just trying to convert local files on your system, tell us about the file type of the file you're trying to convert, & the target type of the converted file you want.  What you've posted so far is too vague for anybody to investigate your problem.

**Comment by paulrouget** [comment]
> "Invalid audio stream. Exactly one MP3 audio stream is required."

This has been solved, it's coming in an upcoming update (8.2.2.4). It's gonna take a few days before it auto update!

**Reply by mackfire** [reply]
Great!  Thanks for the update!
